src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -44,12 +45,18 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
   ArtworkSidebarCommercialContainerProps,
   ArtworkSidebarCommercialContainerState
 > {
+  firstAvailableEcommerceEdition(edition_sets) {
+    return find(edition_sets, edition => {
```

Comment:
This is now available directly on `Array`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
Nice! :+1: